### PR TITLE
flaky: Disable kvstore test

### PR DESCRIPTION
**Pull request by larroy**
## Description ##
Disable failing test on kvstore

https://github.com/apache/incubator-mxnet/issues/11801

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1011,6 +1011,9 @@ try {
         }
       }
     },
+    /*  Disabled due to master build failure:
+     *  http://jenkins.mxnet-ci.amazon-ml.com/blue/organizations/jenkins/incubator-mxnet/detail/master/1221/pipeline/
+
```
Comment (lebeg) [inline Jenkinsfile]: I think you should add a link to the tracking issue here

Comment (marcoabreu) [reply Jenkinsfile]: Exactly. Please also link the issue here in this PR so it pops up as cross reference

Comment (larroy) [reply Jenkinsfile]: Done

**Comment by larroy** [comment]
@szha please merge

**Pull request merged**